### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by eakaraman** [kirainmoe/hasee-tongfang-macos#103] [Q&A] [answered]
Hi, 
My laptop's model is GK5VR0V. 
Is my laptop applicable?

Thanks.

**Comment by kirainmoe** [comment, marked as answer]
Unfortunately I could not find any info about the model. If this model uses 9th (or below) gen Intel processor, it is supported by this repo. 10th gen and above platforms are not supported currently. However, some components in this repo is universal for Tongfang laptops.
